content/6_Advanced/Critical.mdx

```diff
@@ -10,17 +10,318 @@ frequency: 1
 
 ## Paths
 
-CSES Critical Cities https://cses.fi/problemset/task/1703/
+<FocusProblem problem="critical" />
 
-Dominator Tree
+<Resources>
+	<Resource
+		source="Wiki"
+		title="Dominator"
+		url="https://en.wikipedia.org/wiki/Dominator_(graph_theory)"
+		starred
+	>
+  	Wiki Definition
+ 	</Resource>
+	<Resource
+		source="Blog"
+		title="Dominator Tree"
+		url="https://tanujkhattar.wordpress.com/2016/01/11/dominator-tree-of-a-directed-graph/"
+	>
+    Well-covered article
+    </Resource>
+	<Resource
+		source="Blog"
+		title="Visualizing Dominators"
+		url="https://sbaziotis.com/compilers/visualizing-dominators.html"
+		starred
+	>
+    Well-covered article
+    </Resource>
+</Resources>
 
-https://tanujkhattar.wordpress.com/2016/01/11/dominator-tree-of-a-directed-graph/
+### Solution
```

Comment:
make all these headers one less deep (all the way down to line 170 or so)

Reply:
done